playground/worker/worker-plugin-state-plugin.js

```diff
@@ -0,0 +1,15 @@
+export default () => {
+  let count = 0
+  return {
+    name: 'plugin-for-plugin-state',
+    transform(code, id) {
+      if (id.includes('worker/modules/test-state.js')) {
+        count++
```

Comment:
The additions to the integration tests is to highlight how plugins will work when built sequentially vs in parallel.

The idea being: 
- Parallel builds will have a new instance of this plugin every time and so this count increment will only happen once per build. Highlighting a simple level of isolation when building in parallel
- Serial builds will have the same instance of this plugin every time and so this count will increment every time a worker includes the referenced file. Highlighting some potential for state pollution or undesired behaviour.

Let me know if you have alternative ideas of how we could simply and effectively test this 🙇